src/backend/server.js

```diff
@@ -1,28 +1,32 @@
 import express from 'express';
 import doctorOrders from './routes/doctorOrders.js';
 const app = express();
-const port = process.env.PORT || 5051;
```

Comment:
This `server.js` file shouldn't be here (nor should `doctorOrders.js`, really), this PR should base itself off of the `remove_js_files` branch.  Or this can be all fixed when conflicts arise after the remove js PR is merged.  Either way, the js file should be removed.